test/dotnet-trx-merge.UnitTests/Fixtures/Merge/DuplicateTestIdAllPass.trx

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
these are samples files I created from my real files. We need additional ones that have failures in them, so we can test the fix actually works for tests that failed in the first run and passed in the second run

Reply:
I've added the additional tests